source/includes/_authenticated_api_members.md

```diff
@@ -100,9 +100,33 @@ Permanently deletes a member with specified id and all associated records includ
 
 `DELETE /api/v1/members/123`
 
-Deletes are synchronous and may take several seconds depdending on how many resources are owned by the member. 
+Deletes are synchronous and may take several seconds depending on how many resources are owned by the member.
 
-Ownership of Petitions and Events created by the deleted member will be re-assigned to the user account specified in the organisation's settings. 
+Ownership of Petitions and Events created by the deleted member will be re-assigned to the user account specified in the organisation's settings.
+
+
+
+### Anonymize
+
+> POST response body for Anonymize
+
+```json
+{
+  "member": {
+    "deleted": true,
+    "id": 123,
+    "email": "foo@bar.com"
+  }
+}
+```
+
+Anonymizes the member with specified id. This permanently deletes the member and all associated records including petitions, signatures, events. However, petition signatures from the anonymized member are preserved, anonymized, on petition data exports, signature counts, and petition letters. The method of anonymization is configured in organisation name privacy settings.
```

Comment:
Petitions and events are not deleted. They are reassigned.

Reply:
`deletes the member and all associated signatures`?